packages/ifpack2/src/Ifpack2_Details_Chebyshev_decl.hpp

```diff
@@ -64,6 +66,48 @@ template<class TpetraOperatorType>
 class ChebyshevKernel; // forward declaration
 #endif // DOXYGEN_SHOULD_SKIP_THIS
 
+template<class ScalarType, const bool lapackSupportsScalarType = LapackSupportsScalar<ScalarType>::value>
+struct LapackHelper{
+  static
+  ScalarType
+  tri_diag_spectral_radius(Teuchos::ArrayRCP<typename Teuchos::ScalarTraits<ScalarType>::magnitudeType> diag,
```

Comment:
Can the LapackHelper struct be moved out of this `decl` file and into `Ifpack2_Details_Chebyshev_def.hpp`?

Reply:
Done